Project/Wishlist/WishListViewController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  TabOneViewController.swift
+//  autolayout
+//
+//  Created by Emmanuel Velez Agudelo on 29/10/18.
+//  Copyright © 2018 Emmanuel Velez Agudelo. All rights reserved.
+//
+
+import Foundation
+import UIKit
+class WishListViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Setting color.
+        self.title = "WishList"
```

Comment:
`self` keyword should be used only when strictly needed. In this case, just `title = "WishList"` would do the trick.